provided.al2/graalvm/java17/cookiecutter-aws-sam-graalvm-gradle/{{cookiecutter.project_name}}/build-image.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 set -e
 
-docker build -t al2-graalvm:gradle
\ No newline at end of file
+docker build -t al2-graalvm:gradle .
```

Comment:
Do we want to add this `build-image.sh` file into other GraalVM templates? I saw it is only in java17 ones.

Reply:
I believe this change is needed in all four `build-image.sh` files.

Reply:
note: the two under `maven` are correct. The one for java11 gradle needs to be fixed

Reply:
@hawflau So this is fixing the only problematic one then?

Reply:
yep